plugins/pulp_docker/plugins/models.py

```diff
@@ -284,6 +283,32 @@ def from_json(cls, manifest_json, digest):
                    amd64_digest=amd64_digest,
                    amd64_schema_version=amd64_schema_version)
 
+    @staticmethod
+    def validate_json(manifest_json):
+        """
+        Validate a manifest list json file.
+
+        raises: PulpCodedValidationException DKR1011 if invalid JSON
+        raises: PulpCodedValidationException DKR1012 if json contains invalid mediaType
+        raises: PulpCodedValidationException DKR1014 if any of the listed Manifests contain invalid
+                                             mediaType
+        """
+        try:
+            manifest = json.loads(manifest_json)
+        except ValueError:
+            raise PulpCodedValidationException(error_code=error_codes.DKR1011)
+
+        if manifest.get('mediaType') != constants.MEDIATYPE_MANIFEST_LIST:
+            raise PulpCodedValidationException(error_code=error_codes.DKR1012,
+                                               mediaType=manifest['mediaType'])
+
+        for image_man in manifest['manifests']:
+            if image_man['mediaType'] not in [constants.MEDIATYPE_MANIFEST_S2,
+                                              constants.MEDIATYPE_MANIFEST_S1]:
+                raise PulpCodedValidationException(error_code=error_codes.DKR1014,
```

Comment:
the error is not rendered properly:
```
[\]
Running...

Task Failed

Manifest List contains Manifest with digest $(digest)s that has an invalid
mediaType.```

It expects just the digest and you are providing mediatype as well